compiler/compiler_test.go

```diff
@@ -28,18 +28,11 @@ type testCase struct {
 func TestCompiler(t *testing.T) {
 	t.Parallel()
 
-	// Check LLVM version.
+	// Determine LLVM version.
 	llvmMajor, err := strconv.Atoi(strings.SplitN(llvm.Version, ".", 2)[0])
```

Comment:
Should we fail test if `llvmMajor < 13`?

Reply:
Why? We're dropping support for LLVM < 13.